loc/forms.py

```diff
@@ -31,11 +31,26 @@ def get_cleaned_dates(self, cleaned_data=None) -> List[datetime.date]:
         return result
 
 
+_ldmeta = models.LandlordDetails._meta
+_ldname = _ldmeta.get_field('name')
+_ldaddress = _ldmeta.get_field('address')
+
+
 class LandlordDetailsForm(forms.ModelForm):
     class Meta:
         model = models.LandlordDetails
         fields = ('name', 'address')
 
+    name = forms.CharField(
+        required=True,
+        max_length=_ldname.max_length,
+        help_text=_ldname.help_text)
+
+    address = forms.CharField(
+        required=True,
+        max_length=_ldaddress.max_length,
+        help_text=_ldaddress.help_text)
+
```

Comment:
Actually, we might want to just set `blank=False` on the original model fields. If it doesn't require us to specify a default value in the migration, this would be a simpler fix.

Reply:
Ok, I did this in 26f0ea1.